docs/user-guide.md

```diff
@@ -90,6 +90,9 @@ ocpRelease:
   # [Optional]
   cpuArchitecture: cpu-architecture
 # Virtual size of the appliance disk image (at least 150GiB)
```

Comment:
rephrase:
if specified: should be at least 150Gib.
If not specified `your text here`